tensorflow/core/kernels/depthwise_conv_op_gpu.cu.cc

```diff
@@ -1116,6 +1116,7 @@ __launch_bounds__(1024, 2) void DepthwiseConv2dBackpropFilterGPUKernelNHWCSmall(
 
     // Note: the condition to reach this is uniform across the entire block.
     __syncthreads();
+    unsigned active_threads = CudaBallot(0xFFFFFFFF, depth_in_range);
```

Comment:
Can we make 0xFFFFFFFF a constant?  It seems pretty bad if someone were to leave off one of the F's.  Same elsewhere.

Or perhaps the order of args to CudaBallot should be reversed, and 0xFFFFFFFF should be the default for the (now) second argument?

Reply:
cuda_kernel_helper.h now defines CUDA_WARP_ALL for use in the warp-function also defined there.